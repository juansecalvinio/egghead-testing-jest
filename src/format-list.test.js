import formatList from "./format-list";

test('can format a list', () => {
  const formattedList = formatList(
    'Star Wars Names',
    [
      { name: 'Luke Skywalker' },
      { name: 'Obi-Wan Kenobi' },
      { name: 'Han Solo' },
      { name: 'Chewbacca' },
    ],
    'name'
  );
  expect(formattedList).toMatchSnapshot();
})