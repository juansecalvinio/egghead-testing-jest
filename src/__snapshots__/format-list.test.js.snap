exports[`test can format a list 1`] = `
"These are items in the Star Wars Names:
 - Luke Skywalker
 - Obi-Wan Kenobi
 - Han Solo
 - Chewbacca"
`;
